core/container/intrusive/list/intrusive_list.odin

```diff
@@ -104,12 +136,26 @@ remove_by_proc_contextless :: proc(list: ^List, to_erase: proc "contextless" (^N
 	}
 }
 
+/*
+Checks if the given list does not contain any element.
```

Comment:
Minor nitpick: Prefer `Checks whether` instead of `Checks if`, because the latter says the check is conditional on the list not containing any element, rather than whether that's the case.